.github/workflows/markdown_style.yml

```diff
@@ -14,4 +14,6 @@ jobs:
     - name: Install dependencies
       run: npm install `cat npm-requirements.txt`
     - name: Run style checks
-      run: npx remark src/*.md src/*/*.md --frail --rc-path .remarkrc
+      # see: https://stackoverflow.com/a/39991116/5201771
+      # alternative (probably): npx remark ...
+      run: npm run env -- remark src/*.md src/*/*.md --frail --rc-path .remarkrc
```

Comment:
this should work similarly to defining a `scripts` in package.json and then calling `npm run ...`